src/attachment/components/ButtonAttachment.vue

```diff
@@ -8,15 +8,15 @@
       class="cursor-pointer inline-block flex justify-center fit-width"
       :for="id"
     >
-      <FeatherIcon :name="icon" class="inline-block mr-2" />
+      <FeatherIcon :name="icon" class="inline-block mx-1" :size="15"/>
```

Comment:
would be better to also wrap icon in slot so that we can replace icon if needed. the default content will be using existing e.g
``` vue
    <slot name="icon">
        <FeatherIcon :name="icon" class="inline-block mx-1" :size="15"/>
    </slot>
```